File: Arrays/Question 6/is_valid_email.js
function is_valid_email(str){
  let part= str.split('@');

  if (part.length !== 2){
    return false;
  }
  
  first = part[0];
  second = part[1];

  let left = first.split('');
  alpha = 'abcdefghijklmnopqrstuvxxyz';

  for (let i =0; i<left.length; i++){
    if(!(alpha.includes(left[i]))){
      return false;
    }
  }

  let right = second.split('.');
  if(right.length !==2){
    return false;
  }

  else{
    return true;
  }

}

File: Arrays/Question 6/is_valid_email.rb
def is_valid_email(str)
    parts = str.split("@")
  
    if parts.length != 2
      return false
    end
  
    first = parts[0]
    second = parts[1]
    alpha = "abcdefghijklmnopqrstuvwxyz"
  
    first.each_char do |char|
      if !alpha.include?(char)
        return false
      end
    end
  
    if second.split('.').length == 2
      return true
    else
      return false
    end
  end
  